posthog/models/person/person.py

```diff
@@ -32,6 +32,14 @@ def distinct_ids(self) -> List[str]:
             .values_list("distinct_id")
         ]
 
+    def get_distinct_ids_with_version(self) -> Dict[str, int]:
+        return {
+            obj[0]: int(obj[1] or 0)
+            for obj in PersonDistinctId.objects.filter(person=self, team_id=self.team_id)
```

Comment:
Use list unpacking to make code clearer.

E.g. `for distinct_id, version in ...`